components/basic-filters/src/main/java/org/datacleaner/beans/filter/NullCheckFilter.java

```diff
@@ -132,12 +132,12 @@ public Query optimizeQuery(Query q, NullCheckCategory category) {
                 }
                 q.where(column, OperatorType.DIFFERENT_FROM, null);
                 if (considerEmptyStringAsNull && col.getDataType() == String.class) {
-                    q.where(column, OperatorType.DIFFERENT_FROM, "");
+                    q.where(column, OperatorType.LIKE, "%");
```

Comment:
Wait, what? That sounds extremely weird. While the Oracle database maybe isn't the most standards compliant of the bunch, I doubt it would make empty strings NULL unless something specific was put in the column definition.

Could that be a bug in DataCleaner/MetaModel?